File: java/org.kevoree.library.java.javaNode/src/main/java/org/kevoree/library/java/KevoreeThreadGroup.java
package org.kevoree.library.java;

import org.kevoree.log.Log;

/**
 * Created by duke on 9/26/14.
 */
public class KevoreeThreadGroup extends ThreadGroup{

    private String n;

    public KevoreeThreadGroup(String n) {
        super(n);
        this.n = n;
    }

    public void uncaughtException(Thread t, Throwable e) {
        Log.error("Uncatched exception into Kevoree component {}", e, n);
    }
}

File: java/org.kevoree.library.java.javaNode/src/main/java/org/kevoree/library/java/KevoreeThreadGroup.kt
package org.kevoree.library.java

import org.kevoree.log.Log

/**
 * Created by duke on 8/7/14.
 */

class KevoreeThreadGroup(val n: String) : ThreadGroup(n) {


    override fun uncaughtException(t: Thread, e: Throwable) {
        Log.error("Uncatched exception into Kevoree component {}", e, n);
        super<ThreadGroup>.uncaughtException(t, e)
    }


}